pkgs/development/libraries/boost/1.76.nix

```
@@ -0,0 +1,14 @@
+{ callPackage, fetchurl, fetchpatch, ... } @ args:
+
+callPackage ./generic.nix (args // rec {
+  version = "1.76.0";
+
+  src = fetchurl {
+    urls = [
+      "mirror://sourceforge/boost/boost_1_76_0.tar.bz2"
+      "https://dl.bintray.com/boostorg/release/1.76.0/source/boost_1_76_0.tar.bz2"
+    ];
+    # SHA256 from http://www.boost.org/users/history/version_1_76_0.html
+    sha256 = "0hcc661savk32hx65997p0ss1awj6ala4cmz4w7lbi42x6k7nfgh";
+  };
+})
```

```suggestion
{ callPackage, fetchurl, fetchpatch, ... } @ args:

callPackage ./generic.nix (args // rec {
  version = "1.76.0";

  src = fetchurl {
    urls = [
      "mirror://sourceforge/boost/boost_${builtins.replaceStrings ["."] ["_"] version}.tar.bz2"
      "https://dl.bintray.com/boostorg/release/${version}/source/boost_${builtins.replaceStrings ["."] ["_"] version}.tar.bz2"
    ];
    # SHA256 from http://www.boost.org/users/history/version_1_76_0.html
    sha256 = "953db31e016db7bb207f11432bef7df100516eeb746843fa0486a222e3fd49cb";
  };
})
```

updated from version 1.75.